tests/Functional/ContentCallForProposalsRenderTest.php

```diff
@@ -0,0 +1,379 @@
+          'body' => 'Open',
+        ], [
+          'label' => 'Reference',
+          'body' => 'Reference code',
+        ], [
+          'label' => 'Publication date',
+          'body' => $publication_date->format('d F Y'),
+        ], [
+          'label' => 'Opening date',
+          'body' => $opening_date->format('d F Y'),
+        ], [
+          'label' => 'Deadline model',
+          'body' => 'Permanent',
+        ],
+      ],
+    ];
+    $field_list_assert->assertPattern($details_expected_values, $content_items[0]->getHtml());
+
+    // Assert Deadline model and Deadline dates fields.
+    $deadline_date1 = (clone $static_time)->modify('- 3 days');
+    $node->set('oe_call_proposals_model', 'single_stage');
+    $node->set('oe_call_proposals_deadline', $deadline_date1->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT));
+    $node->save();
+    $this->drupalGet($node->toUrl());
+
+    $deadline_date1->setTimeZone(new \DateTimeZone('Australia/Sydney'));
+    $details_expected_values['items'][0] = [
+      'label' => 'Status',
+      'body' => 'Closed',
+    ];
+    $details_expected_values['items'][4] = [
+      'label' => 'Deadline model',
+      'body' => 'Single-stage',
+    ];
+    $details_expected_values['items'][5] = [
+      'label' => 'Deadline date',
+      'body' => $deadline_date1->format('d F Y, H:i (T)'),
+    ];
+    $field_list_assert->assertPattern($details_expected_values, $content_items[0]->getHtml());
+
+    $deadline_selector = '//*[text() = "Deadline date"]/following-sibling::dd[1]/div[@class="ecl-u-type-strike"]';
+    $this->assertSession()->elementExists('xpath', $deadline_selector);
+    $this->assertEquals($deadline_date1->format('d F Y, H:i (T)'), $content_items[0]->find('xpath', $deadline_selector)->getText());
+
+    // Assert Deadline model and multiple Deadline date fields.
+    $deadline_date1 = (clone $static_time)->modify('- 3 days');
+    $deadline_date2 = (clone $static_time)->modify('+ 5 days');
+    $node->set('oe_call_proposals_model', 'two_stage');
+    $node->set('oe_call_proposals_deadline', [
+      $deadline_date1->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+      $deadline_date2->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+    ])->save();
+    $this->drupalGet($node->toUrl());
+
+    $deadline_date1->setTimeZone(new \DateTimeZone('Australia/Sydney'));
+    $deadline_date2->setTimeZone(new \DateTimeZone('Australia/Sydney'));
+    $details_expected_values['items'][0] = [
+      'label' => 'Status',
+      'body' => 'Open',
+    ];
+    $details_expected_values['items'][4] = [
+      'label' => 'Deadline model',
+      'body' => 'Two-stage',
+    ];
+    $expected_deadline_dates = $deadline_date1->format('d F Y, H:i (T)') . $deadline_date2->format('d F Y, H:i (T)');
+    if (version_compare(PHP_VERSION, '7.3') < 0) {
+      $expected_deadline_dates = $deadline_date1->format('d F Y, H:i (T)') . "\n" . $deadline_date2->format('d F Y, H:i (T)');
```

Comment:
This is quite strange, can we explain why this happens?

Reply:
I've faced with this issue a few times. As a generic fix https://github.com/openeuropa/oe_theme/pull/629/commits/504bac8087d10fe7811c0e4c21f62788dda63c4e can be used, but it is in EWPP-402.

Reply:
But the solution you send is a QOL improvement so that normalizes whitespace when using the text() method on a crawler. That doesn't explain why we need different assertions for different PHP versions. I don't see this issue anywhere else, not even in the examples on your generic fix. 
As it is, this could be a problem with the symfony crawler, but it could be we are rendering different things on different php versions, and that is something we should fix.

Reply:
Explanation is in the ticket EWPP-437. 

In the EWPP-402 in some cases issue was resolved by updating templates - new line characters have been removed for templates in oe_theme, in some cases patch helped.

I would prefer to keep current solution now or please suggest another option.